packages/babel-plugin-proposal-optional-chaining/src/index.js

```diff
@@ -237,6 +240,12 @@ export default declare((api, options) => {
               replacementPath.get("alternate"),
             );
           }
+
+          needsRequeue = true;
+        }
+
+        if (needsRequeue) {
+          parentPath.requeue();
```

Comment:
Not a blocker, but we should figure out why the test passes without requeuing parent path on `@babel/traverse` 7.12.3.

Reply:
For sure, in the middle of digging actually

Reply:
It looks like: https://github.com/babel/babel/pull/12302, still digging